src/codewhisperer/commands/basicCommands.ts

```diff
@@ -145,9 +145,10 @@ export const selectCustomizationPrompt = Commands.declare(
 
 export const reconnect = Commands.declare(
     { id: 'aws.codewhisperer.reconnect', compositeKey: { 1: 'source' } },
-    () => async (_: VsCodeCommandArg, source: CodeWhispererSource) => {
-        await AuthUtil.instance.reauthenticate()
-    }
+    () =>
+        async (_: VsCodeCommandArg, source: CodeWhispererSource, addMissingScopes: boolean = false) => {
```

Comment:
thanks for updating the signature! My concern though is that we might need something like
```
if (typeof addMissingScopes !== 'boolean') {
...
}
```

I see this command is referenced in nodes in package.json. If they are called from a package.json menu item, wouldn't this value have the potential to be something else?

Reply:
Ah I see, in the package.json uses the `addMissingScopes` arg doesn't get overwritten in any way and will always use the default value.

But this behavior could change on the vscode side, so I'll add the check you mentioned above.